src/test/crimson/seastore/test_object_data_handler.cc

```diff
@@ -418,3 +425,14 @@ TEST_F(object_data_handler_test_t, multiple_split) {
     read(0, 128<<10);
   });
 }
+
+INSTANTIATE_TEST_SUITE_P(
+  object_data_handler_test,
+  object_data_handler_test_t,
+  ::testing::Values (
+    "segmented",
+    "circularbounded"
+  )
+);
+
+
```

Comment:
See comment https://github.com/ceph/ceph/pull/53232#discussion_r1319422778

How about merging the two PRs enabling RBM tests?